platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/utils/SnapshotterIdlingResource.java

```diff
@@ -0,0 +1,38 @@
+package com.mapbox.mapboxsdk.testapp.utils;
+
+import android.support.test.espresso.IdlingResource;
+
+import com.mapbox.mapboxsdk.testapp.activity.render.RenderTestActivity;
+
+public class SnapshotterIdlingResource implements IdlingResource {
```

Comment:
In https://github.com/mapbox/mapbox-gl-native/pull/9576/commits/c9c2e96240b17ce4bd32729a66cdd3ff213aa313 I'm taking a push approach which significantly speeds up the tests since it idles shorter before being ready. Might be beneficial here?